File: core/src/com/mygdx/game/model/Shotgun.kt
package com.mygdx.game.model

import com.mygdx.game.settings.SHOTGUN_BULLETS_IN_CHAMBER
import com.mygdx.game.settings.SHOTGUN_RELOAD_TIME

class Shotgun : Weapon(SHOTGUN_RELOAD_TIME, SHOTGUN_BULLETS_IN_CHAMBER, ProjectileType.SHOTGUN)

File: server/models/weapons/Shotgun.js
const constants = require('../../settings/constants');
const Weapon = require('./Weapon');
const ProjectileType = require('../projectiles/ProjectileType');

class Shotgun extends Weapon {
    constructor() {
        super(constants.SHOTGUN_RELOAD_TIME, constants.SHOTGUN_BULLETS_IN_CHAMBER,
            constants.SHOTGUN_MAGAZINE_RELOAD_TIME, ProjectileType.SHOTGUN)
    }
}

module.exports = Shotgun